tests/Cookie/CookieJarTest.php

```diff
@@ -37,6 +37,11 @@ public function testCreatesFromArray()
         $this->assertCount(2, $jar);
     }
 
+    public function testEmptyJarIsCountable()
+    {
+        self::assertCount(0, new CookieJar());
```

Comment:
You wrote:
> So I want to check that CookieJar does not throws the warning when calling count

But actually you
> you check that `count == 0`

So nothing changes in my mind if you will check implementing of the \Countable interface^ because it guarantees that calling `count` won't fail.

But it's kind of a holywar... =)